apps/dialtone-documentation/docs/.vuepress/exampleComponents/ExampleModal.vue

```diff
@@ -15,114 +15,85 @@
         />
       </dt-select-menu>
     </div>
-    <button
-      class="d-btn d-btn--primary d-btn--sm d-fl-none"
-      type="button"
-      @click="openModal"
+    <dt-button
+      class="d-fl-none"
+      size="sm"
+      @click="isOpen = !isOpen"
```

Comment:
This is more readable this way IMO

```suggestion
      @click="updateShow(true)"
```